javascript/index.js

```diff
@@ -14,52 +13,93 @@ let milUni = document.getElementById('milUni');
 let splits = document.getElementById('splits');
 
 function printTime() {
-  // ... your code goes here
+  setInterval(() => {
+    printMinutes();
+    printSeconds();
+  }, 100)
 }
 
 function printMinutes() {
-  // ... your code goes here
+  minDec.innerText = chronometer.twoDigitsNumber(chronometer.getMinutes())[0];
+  minUni.innerText = chronometer.twoDigitsNumber(chronometer.getMinutes())[1];
 }
 
 function printSeconds() {
-  // ... your code goes here
+  secDec.innerText = chronometer.twoDigitsNumber(chronometer.getSeconds())[0];
+  secUni.innerText = chronometer.twoDigitsNumber(chronometer.getSeconds())[1];
 }
 
+
 // ==> BONUS
 function printMilliseconds() {
-  // ... your code goes here
+
+  
 }
 
 function printSplit() {
   // ... your code goes here
+  splits.innerHTML += `<div>${chronometer.splitClick()}</div>`;
 }
 
 function clearSplits() {
-  // ... your code goes here
+
+  splits.innerHTML = '';
+
 }
 
 function setStopBtn() {
-  // ... your code goes here
+
+  chronometer.setStopBtn();
```

Comment:
There is no method called `setStopBtn` in the Chronometer class. 
This call here does not refer to anything the chronometer instance can do. 

You are already setting the functionality of this method in your event listener below. You only need to write it here instead and call this `setStopBtn()` method in the event listener instead. 

 ```suggestion
  btnLeft.className = 'btn stop';
  btnLeft.innerText = 'STOP';
```